pyomo/repn/plugins/gams_writer.py

```diff
@@ -868,6 +868,7 @@ def _write_model(self,
 'PYOMO': {'LP','MIP','RMIP','NLP','MCP','MPEC','RMPEC','CNS','DNLP','RMINLP','MINLP'},
 'QUADMINOS': {'LP'},
 'SBB': {'MINLP','MIQCP'},
+'SHOT': {'MINLP','MIQCP'},
```

Comment:
Could you move this down a couple lines (after SCIP) to keep the solvers in alphabetical order?

Reply:
Sure, my bad.